test/suites/dev/moonbase/test-pov/test-max-pov.ts

```diff
@@ -0,0 +1,132 @@
+import "@moonbeam-network/api-augment";
+import { beforeAll, deployCreateCompiledContract, describeSuite, expect } from "@moonwall/cli";
+import { ALITH_ADDRESS, createEthersTransaction } from "@moonwall/util";
+import { type Abi, encodeFunctionData } from "viem";
+
+describeSuite({
+  id: "D012703",
+  title: "PoV size test - approaching maximum limit",
+  foundationMethods: "dev",
+  testCases: ({ context, it, log }) => {
+    let storageFillerAddress: `0x${string}`;
+    let storageFillerAbi: Abi;
+
+    // Target a PoV size of approximately 7.5MB
+    const TARGET_POV_MB = 7.5;
```

Comment:
Currently is 3,75MB

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3198/commits/1aa39ac2dbbb5cc5d4425a2196003ca29691f140